runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineDebugOptions.java

```diff
@@ -191,6 +191,18 @@ public Dataflow create(PipelineOptions options) {
   void setOverrideWindmillBinary(String value);
 
   /**
+   * Custom windmill service endpoint.
+   */
+  @Description("Custom windmill service endpoint.")
```

Comment:
The worker code currently uses the presence of this flag to decide to use windmill service, so it is correct to have no default